### PR TITLE
BAU - Displaying BST/GMT timestamp for the EAD document

### PR DESCRIPTION
**Pull request by PauloPortugal**

### CODE REVIEW

File: test/unit/uk/gov/hmrc/exports/models/ead/parsers/MrnStatusParserTestData.scala [hunk 1]
```diff
@@ -31,7 +31,7 @@ object MrnStatusParserTestData {
     <p:DeclarationStatusDetails>
       <p:Declaration>
         <p:AcceptanceDateTime>
-          <p1:DateTimeString formatCode="304">20190702110757Z</p1:DateTimeString>
+          <p1:DateTimeString formatCode="304">20190102110757Z</p1:DateTimeString>
```
Comment (PauloPortugal) [inline test/unit/uk/gov/hmrc/exports/models/ead/parsers/MrnStatusParserTestData.scala:34]: This is January date, so this time should always be `11:07am`

Comment (SteveSugden) [inline test/unit/uk/gov/hmrc/exports/models/ead/parsers/MrnStatusParserTestData.scala:34]: You change the date to be in the summer?

File: test/unit/uk/gov/hmrc/exports/models/ead/parsers/MrnStatusParserSpec.scala [hunk 2]
```diff
@@ -28,9 +28,9 @@ class MrnStatusParserSpec extends WordSpec with MustMatchers with OptionValues {
       mrnStatus.eori mustBe "GB123456789012000"
       mrnStatus.versionId mustBe "1"
       mrnStatus.declarationType mustBe "IMZ"
-      mrnStatus.acceptanceDateTime mustBe Some("02 July 2019 at 11:07am")
-      mrnStatus.receivedDateTime mustBe "02 July 2019 at 11:08am"
-      mrnStatus.releasedDateTime mustBe Some("02 July 2019 at 11:09am")
+      mrnStatus.acceptanceDateTime mustBe Some("02 January 2019 at 11:07am")
+      mrnStatus.receivedDateTime mustBe "02 July 2019 at 12:08pm"
+      mrnStatus.releasedDateTime mustBe Some("02 July 2019 at 12:09pm")
```
Comment (PauloPortugal) [inline test/unit/uk/gov/hmrc/exports/models/ead/parsers/MrnStatusParserSpec.scala:33]: These dates were created during BST. Their UTC time is `11:08am` and `11:09am` respectively, so when displaying this to the user/FE we need to adjust the time by one hour.

File: test/unit/uk/gov/hmrc/exports/models/ead/parsers/MrnStatusParserTestData.scala (hunk 1 above)
Comment (SteveSugden) [reply test/unit/uk/gov/hmrc/exports/models/ead/parsers/MrnStatusParserTestData.scala:34]: Winter, even.   So 11:07 in winter must be displayed as 11:07?
What should that time be in summer?  Can we have a test for that?

Comment (PauloPortugal) [reply test/unit/uk/gov/hmrc/exports/models/ead/parsers/MrnStatusParserTestData.scala:34]: to be in Winter actually. I am changing from July to January, in order to provide an example of a time that should not change.
```
old: 20190702110757Z
new: 20190102110757Z
```

Comment (SteveSugden) [reply test/unit/uk/gov/hmrc/exports/models/ead/parsers/MrnStatusParserTestData.scala:34]: Gotcha!

Comment (PauloPortugal) [reply test/unit/uk/gov/hmrc/exports/models/ead/parsers/MrnStatusParserTestData.scala:34]: The test has been updated to reflect that: check `MrnStatusParserSpec.scala`

```
 mrnStatus.acceptanceDateTime mustBe Some("02 January 2019 at 11:07am")
 mrnStatus.receivedDateTime mustBe "02 July 2019 at 12:08pm"
 mrnStatus.releasedDateTime mustBe Some("02 July 2019 at 12:09pm")
```

Comment (SteveSugden) [reply test/unit/uk/gov/hmrc/exports/models/ead/parsers/MrnStatusParserTestData.scala:34]: Yes, I get it now.  Merge away.

**Pull request merged**